packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -1315,6 +1342,40 @@ internal class DdSdkTest {
         }
     }
 
+    @Test
+    fun `𝕄 send long tasks 𝕎 frame time is over threshold() { never vitals frequency update }`(
+        @LongForgery(min = 0L) timestampNs: Long,
+        @LongForgery(min = ONE_HUNDRED_MILLISSECOND_NS, max = 5 * ONE_SECOND_NS) threshold: Long,
+        @LongForgery(min = 1, max = ONE_SECOND_NS) frameDurationOverThreshold: Long,
+        @Forgery configuration: DdSdkConfiguration
+    ) {
+        // Given
+        val bridgeConfiguration = configuration.copy(
+            vitalsUpdateFrequency = "NEVER",
+            longTaskThresholdMs = (threshold / 1_000_000).toDouble(),
+        )
+        val frameDurationNs = threshold + frameDurationOverThreshold
+
+        // When
+        testedBridgeSdk.initialize(bridgeConfiguration.toReadableJavaOnlyMap(), mockPromise)
+
+        // Then
+        argumentCaptor<Choreographer.FrameCallback> {
+            verify(mockChoreographer).postFrameCallback(capture())
+
+            // When
+            firstValue.doFrame(timestampNs)
+            firstValue.doFrame(timestampNs + frameDurationNs)
+
+            // then
+            verify(mockRumMonitor._getInternal()!!).addLongTask(
```

Comment:
we should also check here that there was no interaction with `updatePerformanceMetric` method, because we are using `NEVER`. Or maybe create a test which is checking only `updatePerformanceMetric` call in case of `NEVER`? Cannot find such.